<launch>
    <!-- Default arguments -->
    <arg name="drone_id1" default="kingfisher"/>
    <arg name="drone_id2" default="kingfisher1"/>
    <arg name="target_x" default="0.0"/>
    <arg name="target_y" default="0.0"/>
    <arg name="target_z" default="5.0"/>
    <arg name="kp" default="0.1"/>
    <arg name="ki" default="0.01"/>
    <arg name="kd" default="0.05"/>

    <!-- Node for Command-->
    <group ns="BEP">
        <node name="command_node"
              pkg="drone_controller"
              type="command_node"
              output="screen">
            <param name="drone_id1" value="$(arg drone_id1)"/>
            <param name="drone_id2" value="$(arg drone_id2)"/>
            <param name="target_x" value="$(arg target_x)"/>
            <param name="target_y" value="$(arg target_y)"/>
            <param name="target_z" value="$(arg target_z)"/>
            <param name="kp" value="$(arg kp)"/>
            <param name="ki" value="$(arg ki)"/>
            <param name="kd" value="$(arg kd)"/>
        </node>
    </group>
</launch>
